Frameworks/Foundation/NSURLProtocol_WinHTTP.mm

```diff
@@ -251,7 +251,7 @@ - (void)startLoading {
 
         NSError* error = nil;
         try {
-            auto httpRequestMessage = _requestMessageForNS(_request);
+            auto httpRequestMessage = _requestMessageForNS(self.request);
```

Comment:
If that's the case, we should at least lift these out to locals then. Otherwise, each hit is a `[[_request retain] autorelease]`